### ISSUE TITLE
More bookshelves menu not keyboard operable

### ISSUE THREAD

**Issue by rkingett** [mouse-reeve/bookwyrm#350]
On the bookwyrm.social website, all instances of the, more bookshelves, menu cannot be used via the keyboard alone, running NVDA on Windows 10.

I tried to do some research on coding an accessible dropdown menu, but everything I've found applies to navigation. Ironically, Goodreads has an accessible dropdown menu... even if it is a bit clunky to use.

Would it be too visually cluttered if we moved the, currently reading, and, already read, bookshelf buttons out of that menu and placed them by, want to read?

Or we could have, currently reading, next to the want to read button, since a user can just rate a book if they want to add it to their already read shelf, right?

I'm thinking it might be, but I can't find a good resource online how to code these to make them keyboard friendly.

To open the menu, I have to click it with my mouse. Novice screen reader users may not know how to snap the mouse to the currently focused element.

**Comment by mouse-reeve** [comment]
Yeah, thank you, this menu is really broken, both visually and for screen reader. There's a different menu on the shelf page, in the table of books, which lets you select a shelf to move a book to (or un-shelve it) -- I think that menu is accessible, is it working for you?

**Comment by rkingett** [comment]
The menu found in the tables section of shelves is very accessible with the keyboard! So maybe take from that menu?

**Comment by mouse-reeve** [comment]
okay great! I can use that as a reference implementation
